globeandmail/aws-codepipeline-lambda
# v1.6 Use CodeBuild Project Version 1.7 to Expose Environment Variable DS_DEPLOY_GITHUB_TOKEN_SECRETS_ID

#### PR description

What is it for?
The PR is to upgrade the codebuild project module version to 1.7 to expose the secrets-manager environment variable DS_DEPLOY_GITHUB_TOKEN_SECRETS_ID.
Also added release note to README.md.

#### Testing instructions

- Test the new version of the code pipeline module in any Sophi pop lambda code pipeline. Remember to create a new pipeline for test.
- Verify a successful release in the test lambda code pipeline.

#### JIRA ticket information

Story: [DSOPS-81](https://jira.theglobeandmail.com/browse/DSOPS-81)
